src/main/resources/io/jenkins/plugins/PurgeBuildQueueAction/action.jelly

```diff
@@ -24,10 +24,11 @@
 
 <?jelly escape-by-default='false'?>
 <j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
-  <l:hasPermission permission="${app.ADMINISTER}">
+  <l:hasAdministerOrManage>
     <l:task icon="symbol-trash-outline plugin-ionicons-api" confirmationMessage="${%Purge the build queue?}"
             href="purge-build-queue/purge/" post="true" requiresConfirmation="true"
-            title="${%Purge Build Queue}" permissions="${it.ADMINISTER}">
```

Comment:
`permissions="${it.ADMINISTER}"`  was the bug that caused this not to be compatable.

`it` is not `Jenkins` in all current versions as such `it.ADMINISTER` would return `null` and then the permission would not be checked.
In Future version `it` will be an instance of Jenkins and as such this becomes valid, but it is a single permission when `permissions` is expecting an array.  

Given this is already protected by the permission check in `layout` we can simply rmeove the extra check.